smarts/zoo/agent_spec.py

```diff
@@ -67,13 +67,9 @@ class AgentSpec:
     """Parameters to be given to `AgentSpec.agent_builder` (default None)"""
 
     observation_adapter: Callable = lambda obs: obs
-    """An adaptor that allows shaping of the observations (default lambda obs: obs)"""
     action_adapter: Callable = lambda act: act
-    """An adaptor that allows shaping of the action (default lambda act: act)"""
     reward_adapter: Callable = lambda obs, reward: reward
-    """An adaptor that allows shaping of the reward (default lambda obs, reward: reward)"""
     info_adapter: Callable = lambda obs, reward, info: info
```

Comment:
I think we should add deprecation warnings about these.

Reply:
Added a deprecation warning message.